torch/export/_trace.py

```diff
@@ -435,7 +436,13 @@ def make_argument_spec(node) -> ArgumentSpec:
             return TensorArgument(name=node.name)
         elif isinstance(val, torch.SymInt):
             return SymIntArgument(name=node.name)
+        elif isinstance(val, torch.ScriptObject):
+            return CustomObjArgument(
+                name=node.name, class_fqn=val._type().qualified_name()  # type: ignore[attr-defined]
+            )
         else:
+            # TODO: this branch is likely wrong, all permissible ConstantArgument type
```

Comment:
hmm, I'm not sure what this comment means

Reply:
should raise error if reaching this else branch